sources/packages/forms/src/form-definitions/noticeofassessment.json

```diff
@@ -13305,7 +13504,7 @@
                         "value": ""
                       }
                     ],
-                    "content": "${{((data.disbursement?.disbursement1cspt ? parseFloat(data.disbursement.disbursement1cspt) : 0) + (data.disbursement?.disbursement2cspt ? parseFloat(data.disbursement.disbursement2cspt) : 0)).toFixed(2)}}",
```

Comment:
This is the only change I did as part of the Ecert request file creation, other changes are either id or addons tags generated or people changed the NOA and did not check in to GITHUB